curriculum/challenges/english/02-javascript-algorithms-and-data-structures/debugging/understanding-the-differences-between-the-freecodecamp-and-browser-console.md

```diff
@@ -50,24 +40,26 @@ assert(__helpers.removeWhiteSpace(code).match(/console\.log\(output\)/));
 
 ```js
 // Open your browser console.
-let output = "Get this to log once in the freeCodeCamp console and twice in the browser console";
-// Use console.log() to print the output variable.
+let output = "Get this to show once in the freeCodeCamp console and not at all in the browser console";
 
-// Run the tests to see the difference between the two consoles.
+// Use console.log() to print the output variable
 
-// Now, add console.clear() before your console.log() to clear the browser console, and pass the tests.
+// View the freeCodeCamp console and the browser console
```

Comment:
This will require an addition to the comment-dictionary, yes?

Reply:
nice catch - do you think it would be better to just add it to the dictionary or try to adjust this to not add a new comment? I could probably make it work by removing that comment, and maybe adding something to the instructions.

Reply:
In general, I am happier with **not** adding to the comment dictionary. So, if you have an idea for this, then by all means 🚢